widgetssdk/src/main/java/com/glia/widgets/helper/Logger.kt

```diff
@@ -118,6 +120,32 @@ internal object Logger {
         })
     }
 
+    @JvmStatic
+    fun logDeprecatedMethodUse(tag: String, methodName: String) {
+        val logMessage = DEPRECATED_METHOD_CALL_LOG_MESSAGE + methodName
+
+        if (loggingAdapters.isEmpty() && BuildConfig.DEBUG) {
+            Log.d(tag, logMessage)
+        }
+
+        loggingAdapters.forEach(Consumer { loggingAdapter: LoggingAdapter ->
+            loggingAdapter.log(LogLevel.DEBUG, tag, logMessage, concatGlobalMeta(null))
```

Comment:
Out of curiosity why debug logs did not work before?

Reply:
@gugalo I will replace `debug` log level with `info`.
The same as I did for Core SDK.
`debug` is not showing up in Kibana.

Reply:
Debug is not supposed to show up in kibana, we were sending all but debug level logs to kibana

Reply:
@gugalo FYI, changed level to `info` now.